src/controllers/controlpickermenu.cpp

```diff
@@ -802,6 +802,20 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addDeckControl("waveform_zoom", tr("Waveform Zoom"), tr("Waveform zoom"), guiMenu);
     addDeckControl("waveform_zoom_down", tr("Waveform Zoom In"), tr("Zoom waveform in"), guiMenu);
     addDeckControl("waveform_zoom_up", tr("Waveform Zoom Out"), tr("Zoom waveform out"), guiMenu);
+
+    // Controls to change a deck's star rating
+    QString starsUpTitle = tr("Star Rating Up");
+    QString starsUpDescription = tr("Increase the track rating by one star");
```

Comment:
Just out of curious: 
For my German ears "Upgrades the track rating .. " sounds better. I would use "Increase" for floating point values. 
What do the native speaker say?